Gemfile

```diff
@@ -62,7 +59,7 @@ gem 'font-awesome-sass', '~> 5.13'
 gem 'jquery-easing-rails'
 gem 'jquery_mask_rails', '~> 0.1.0'
 gem 'normalize-rails', '~> 4.1.1'
-gem 'validates_timeliness', '~> 6.0.0.beta2', github: "mitsuru/validates_timeliness", branch: "rails7"
+gem 'validates_timeliness', '> 6.0.0'
```

Comment:
did you check if the gem really has support for this rails version, the part of code calling this may not have spec coverage

Reply:
I checked and it has support for Rails 7.1.1, so I did a patch downgrade.